soh/soh/GameMenuBar.cpp

```diff
@@ -694,6 +694,9 @@ namespace GameMenuBar {
                     UIWidgets::PaddedEnhancementCheckbox("Remember Save Location", "gRememberSaveLocation", true, false);
                     UIWidgets::Tooltip("When loading a save, places Link at the last entrance he went through.\n"
                             "This doesn't work if the save was made in a grotto.");
+                    UIWidgets::EnhancementSliderInt("Fast Forward: %dx", "##FASTFORWARD", "gFrameMult", 1, 8, "", 1);
```

Comment:
Can you add this cvar to the presets as well?  Specifically, add it to the Default section with its lowest value of 1 so that when people pick the Default preset, this is turned off.

Reply:
Changes made